asf_heat_pump_suitability/getters/load_geodata.py

```diff
@@ -27,3 +28,20 @@ def load_df_osopen_uprn(**kwargs) -> pl.DataFrame:
     )
 
     return df
+
+
+def load_gdf_plymouth_heat_network_zones(**kwargs) -> gpd.GeoDataFrame:
+    """
+    Load GeoDataFrame with Plymouth heat network zone polygons from Plymouth City Council (CRS: EPSG:27700).
+
+    Args:
+        **kwargs for `gpd.read_file()`
+
+    Returns:
+        gpd.GeoDataFrame: polygons of heat network zones in Plymouth.
+    """
+    gdf = base_getters.get_gdf_from_gpkg_s3_path(
+        path=config["data"]["geodata"]["heat_network_zones"]["plymouth"],
+        **kwargs,
+    )
+    return gdf
```

Comment:
With scaling in mind, would it make sense to have the following?
```suggestion
def load_gdf_heat_network_zones(council_or_area="plymouth":str, **kwargs) -> gpd.GeoDataFrame:
    Load GeoDataFrame with Plymouth heat network zone polygons from Plymouth City Council (CRS: EPSG:27700).

    Args:
        council_or_area (str): council or local area considered. Defaults to "plymouth"
        **kwargs for `gpd.read_file()`

    Returns:
        gpd.GeoDataFrame: polygons of heat network zones in Plymouth.
    """
    gdf = base_getters.get_gdf_from_gpkg_s3_path(
        path=config["data"]["geodata"]["heat_network_zones"][council_or_area],
        **kwargs,
    )
    return gdf
```

Reply:
cc: @crispy-wonton

Reply:
I think it makes sense! If you agree @crispy-wonton, I'll make that change

Reply:
Yep looks good. 2 minor things, please could we use `local_authority` as the arg and config names for consistency, and secondly please no default, but instead call `local_authority="Plymouth"` explicitly where the function is used. This is to avoid unintentional silent fails. For example, if you didn't input the LA name and called the function with the Plymouth default but were using it for another LA, you would conduct the spatial join and it would erroneously look like none of the UPRNs fall in a HN zone but the code wouldn't error out.